Firmware/ultralcd.cpp

```diff
@@ -5194,7 +5205,13 @@ static void lcd_main_menu()
     } else if (!Stopped) {
         MENU_ITEM_SUBMENU_P(_i("Preheat"), lcd_preheat_menu);////MSG_PREHEAT c=18
     }
-
+    if (GetPrinterState() < PrinterState::IsSDPrinting && M79_timer_get_status()) {
+        if(GetPrinterState() == PrinterState::IsReady) {
+            MENU_ITEM_TOGGLE_P(_T(MSG_SET_READY), _T(MSG_NO), lcd_printer_status_toggle);
+        } else {
+            MENU_ITEM_TOGGLE_P(_T(MSG_SET_READY), _T(MSG_YES), lcd_printer_status_toggle);
+        }
+    }
```

Comment:
Nitpick, this comment doesnt block this PR... but I think this would be more readable: (it doesnt cost any extra memory)

```cpp
if (M79_timer_get_status()) {
        switch (GetPrinterState())
        {
        case PrinterState::IsReady:
            MENU_ITEM_TOGGLE_P(_T(MSG_SET_READY), _T(MSG_NO), lcd_printer_status_toggle);
            break;
        case PrinterState::NotReady:
        case PrinterState::Idle:
        case PrinterState::SDPrintingFinished:
        case PrinterState::HostPrintingFinished:
            MENU_ITEM_TOGGLE_P(_T(MSG_SET_READY), _T(MSG_YES), lcd_printer_status_toggle);
            break;
        default:
            break;
        }
    }
```